_data/osdf_namespace_metadata/aws-opendata.yaml

```
@@ -0,0 +1,21 @@
+description: |
+**[AWS Open Data](https://aws.amazon.com/opendata/)** hosts publicly accessible datasets covering areas such as earth science, climate, genomics, machine learning, transportation, and economics. The collection includes contributions from a range of organizations, including government agencies, academic institutions, and private companies.
+
+There are currently nearly **700 datasets**, totaling over **100 petabytes of data**.
+
+Browse the full catalog at the **[Registry of Open Data on AWS](https://registry.opendata.aws)**.
+
+organization: Amazon Web Services, Inc. # Name of the organization
+dataVisibility: public # public, or private
+size: # In bytes so I can parse how I want
+bytesXferd: # In bytes so I can parse how I want
+url: https://aws.amazon.com/opendata/ # Organization URL
+fieldOfScience: MULTI/INTERDISCIPLINARY STUDIES. # Top level Field of Science pulled from here https://nces.ed.gov/ipeds/cipcode/browse.aspx?y=55
+numberOfDatasets: 690 # Number of datasets in the namespace
+rank: # Rank of the namespace, 0 will have the lowest priority
+display: true # Whether or not to display this namespace
+name: Amazon Web Services Open Data # Human readable name of the namespace
```

There is a newly added field missing in this PR:
```suggestion
inProgress: false # Whether or not the namespace is in progress
```